resallocwebui/app.py

```diff
@@ -0,0 +1,50 @@
+import os
+from flask import Flask, render_template
+from resallocserver.app import session_scope
+from resallocserver.app import app as application
+from resallocserver.logic import QResources
+from resalloc.helpers import load_config_file
+
+app = Flask(__name__)
+
+
+@app.route('/resources')
+def resources():
+    with session_scope() as session:
+        resources = QResources(session)
```

Comment:
FTR, I think we can keep session_scope() here for now, but long term we should start using equivalent approach to what we have in Copr (flask sqlalchemy, or similar - so we have a session scope per request).